api/users/serializers.py

```diff
@@ -328,6 +335,110 @@ class Meta:
         type_ = 'institutions'
 
 
+class RelatedEducation(JSONAPIRelationshipSerializer):
+    id = ser.CharField(required=False, allow_null=True, source='_id')
+    class Meta:
+        type_ = 'education'
+
+    def get_absolute_url(self, obj):
+        return obj.absolute_api_v2_url
+
+
+class UserEducationRelationshipSerializer(BaseAPISerializer):
+    # TODO - consolodate these into a user relationship serializer?
```

Comment:
There's a lot of repeated code in all of these relationship serializers that could definitely be consolidated